QuickDeployment/QuickDeploymentInternetSetup.md

```diff
@@ -0,0 +1,215 @@
+* `$Credential`: This is the `chocouser` credential used to connect to Nexus that we specified earlier in [Nexus Setup](#nexus-setup). You can generate this ahead of time via a command like `$Credential = Get-Credential`
+* `$ClientSalt`: This is the client-side salt additive we discussed in the [CCM Setup](#ccm-setup) section above.
+* `$ServerSalt`: This is the server-side salt additive we discussed in the [CCM Setup](#ccm-setup) section above.
+
+An example of running this script with the requisite parameters on an endpoint is as follows (to be run from a PowerShell Administrator window):
+
+```powershell
+Set-ExecutionPolicy Bypass -Scope Process -Force; . C:\PATH\TO\ClientSetup.ps1 -Fqdn "'<YOUR_FQDN_HERE>'" -Credential $(Get-Credential) -ClientSalt "'YourSuperSecureSalt1'" -ServerSalt "'YourSuperSecureSalt2'"
+```
+
+This script will accomplish the following:
+
+* Install Chocolatey from the installation script hosted in your internal raw Nexus repository
+* Add the `ChocolateyInternal` source, and enable it for self-service
+* Disable the default `Chocolatey` Community Repository on your endpoints
+* Install your Chocolatey license using the `chocolatey-license` package you created
+* Install the Chocolatey Licensed Extension (without context menus for Package Builder)
+* Install the `ChocolateyGUI` package on the endpoint, for self-service support
+* Install the `chocolatey-agent` package, which supports self-service and CCM communication
+* Enable and disable features related to configuring self-service access on the endpoint
+* Setup the communication channel between the endpoint and CCM, using the correct URL and salts
+* Opt the endpoint into CCM Deployments
+
+Currently, the easiest way to accomplish this on-boarding is to copy this script over to your endpoints, and just run it from there in a PowerShell Administrator window. We are working on a process to make this simpler, and we hope to release that shortly. We will update this document as soon as an alternate method becomes available.
+
+## Jenkins
+
+The Jenkins open source automation server application is installed on QDE, and available locally on the VM at port 8080 (https://localhost:8080). As this is a local website, and accessible over HTTP, it is our recommendation to just utilize the tool locally on the VM itself, instead of making it accessible elsewhere.
+
+If you **must** open up access to Jenkins internally, we advise you to review the following documentation in order to better secure that access:
+* [Jenkins Security Recommendations](https://www.jenkins.io/doc/book/system-administration/security/)
+* [Jenkins SSL Setup by Sam Gleske](http://sam.gleske.net/blog/engineering/2016/05/04/jenkins-with-ssl.html)
+
+> :warning: **WARNING:**
+> As Jenkins is more of a "set-it-and-forget-it" service, we strongly advise you **do not** open up access to the Jenkins web console, but rather just use the web console on the QDE VM itself.
```

Comment:
You can also mention that they can access this on architecture that is on the same network or through a VPN.